CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Print warning about unknown resource state before exiting when execution is interrupted with SIGINT.
+- Add `kubernetes nodegroup create`, `kubernetes nodegroup scale`, and `kubernetes nodegroup delete` commands (EXPERIMENTAL)
```

Comment:
I was thinking that we could mention here that creating a node group with `kubernetes create` is still possible, but maybe it's too verbose for the change log. Up to the author :)